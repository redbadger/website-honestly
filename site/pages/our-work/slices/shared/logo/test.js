// @flow

import React from 'react';
import { shallow } from 'enzyme';

import Logo from '.';

describe('site/pages/our-work/slices/shared/logo', () => {
  it('renders correctly', () => {
    const component = shallow(<Logo src="path/to/image" slice="camden" />);
    expect(component).toMatchSnapshot();
  });
});
