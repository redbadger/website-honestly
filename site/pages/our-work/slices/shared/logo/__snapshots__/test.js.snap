// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`site/pages/our-work/slices/shared/logo renders correctly 1`] = `
<img
  alt=""
  className="camden"
  src="path/to/image"
/>
`;
